pkgs/development/python-modules/pydeps/default.nix

```diff
@@ -10,7 +10,7 @@
 
 buildPythonPackage rec {
   pname = "pydeps";
-  version = "1.10.18";
+  version = "1.10.22";
```

Comment:
It doesn't look like this package is depended on by anything in Nixpkgs (including pip-audit), why bump it here?

Reply:
It doesn't have to be bumped here -> #182931